e2e/cypress/integration/messaging/center_channel_rhs_overlap_spec.js

```diff
@@ -7,24 +7,62 @@
 // - Use element ID when selecting an element. Create one if none.
 // ***************************************************************
 
-// Stage: @prod
 // Group: @messaging
 
+import * as TIMEOUTS from '../../fixtures/timeouts';
+import * as MESSAGES from '../../fixtures/messages';
+
 describe('Messaging', () => {
+    let testTeam;
+    let testUser;
+    let otherUser;
+    let townsquareLink;
+
+    const firstMessage = 'Hello';
+    const message1 = 'message1';
+    const message2 = 'message2';
+    const messageX = 'messageX';
+
+    const messageWithCodeblock1 = '```{shift}{enter}codeblock1{shift}{enter}```{shift}{enter}';
+    const messageWithCodeblockTextOnly1 = 'codeblock1';
+    const messageWithCodeblockIncomplete2 = '```{shift}{enter}codeblock2';
+    const messageWithCodeblockTextOnly2 = 'codeblock2';
+    const messageWithCodeblockIncomplete3 = '```{shift}{enter}codeblock3';
+    const messageWithCodeblockTextOnly3 = 'codeblock3';
+
+    function isMac() {
+        return navigator.platform.toUpperCase().indexOf('MAC') >= 0;
+    }
+
     before(() => {
-        // # Change viewport to tablet
-        cy.viewport('ipad-2');
+        cy.apiAdminLogin();
```

Comment:
`cy.apiAdminLogin();` can be safely removed since up until here the session is of sysadmin.